src/platform_impl/linux/mod.rs

```diff
@@ -1,12 +1,21 @@
 #![cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))]
 
-use std::{collections::VecDeque, env, ffi::CStr, fmt, mem::MaybeUninit, os::raw::*, sync::Arc};
+#[cfg(all(not(feature = "x11"), not(feature = "wayland")))]
+compile_error!("Please select a feature to build for web: `w11`, `wayland`");
```

Comment:
`to build for web`? I don't understand this. Is this a mistake?

Also, you've written `w11` here instead of `x11`.

Reply:
bad copy-paste